modules/porous_flow/doc/content/source/fvkernels/FVPorousFlowMassTimeDerivative.md

```diff
@@ -2,12 +2,14 @@
 
 !syntax description /FVKernels/FVPorousFlowMassTimeDerivative
 
-This `Kernel` implements the weak form of
+This `FVKernel` implements the strong form of
 \begin{equation*}
   \frac{\partial}{\partial t}\left(\phi\sum_{\beta}S_{\beta}\rho_{\beta}\chi_{\beta}^{\kappa}\right)
 \end{equation*}
 where all parameters are defined in the [nomenclature](/nomenclature.md).
 
+!alert note Presently, a first-order accurate implicit Euler time derivative is hard-coded.
```

Comment:
```suggestion
!alert note
Presently, a first-order accurate implicit Euler time derivative is hard-coded.
```
this is not showing currently
https://mooseframework.inl.gov/docs/PRs/26279/site/source/fvkernels/FVPorousFlowMassTimeDerivative.html

Reply:
Oops - I'll fix this!